packages/core/src/MultiFactorCredential.php

```diff
@@ -82,10 +82,6 @@ protected static function newFactory(): Factory
      */
     public function user()
     {
-        $guard = Config::get('auth.defaults.guard');
-        $provider = Config::get('auth.guards.'.$guard.'.provider');
-        $model = Config::get('auth.providers.'.$provider.'.model');
-
-        return $this->belongsTo($model, 'user_id', (new $model())->getKeyName());
+        return $this->belongsTo(LaravelAuth::userModel(), 'user_id');
```

Comment:
Guess there was already an implementation were the user model was determined from the config.